components/kyma-environment-broker/internal/storage/driver/memory/cls_instance.go

```diff
@@ -0,0 +1,65 @@
+package memory
+
+import (
+	"sync"
+
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal"
+	"github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/storage/dberr"
+)
+
+type clsInstances struct {
+	mu sync.Mutex
+
+	data map[clsKey]internal.CLSInstance
+}
+
+type clsKey struct {
```

Comment:
I see that we're building a 1-to-1 mapping of cls instances and the global accounts. What if we are required to create more than one instance for a global account at some point in the future? Would it be too difficult to design a solution supporting more than one instance from the beginning? What do you think?

Reply:
Well, there should be a way to identify _our_ instance. Currently, we are using the `GlobalAccountID` as a unique key to search for. But you are right, we might use a combination of a `GlobalAccountID` + `Region` in the future, so I'll add a `Region` field.

Reply:
Added here: 5b64a86f089473ce2d653af54a4fa9fca965c97b